apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx

```diff
@@ -209,6 +211,26 @@ export const AttributeListItem: FunctionComponent<AttributeListItemPropInterface
                     }
                 />
             </Table.Cell>
+            { deleteAttribute ? (
+                <Table.Cell textAlign="right">
+                    <Popup
+                        trigger={ (
+                            <Icon
+                                link={ true }
+                                className="list-icon"
+                                size="large"
+                                color="grey"
+                                name="trash alternate"
+                                onClick={ deleteAttribute }
+                            />
+                        ) }
+                        position="top center"
+                        content={ "Remove attribute" }
```

Comment:
Lets add i18n strings here.
```suggestion
                        content={ "Remove attribute" }
```

Reply:
Fixed in 5c44499